### PR TITLE
Create sumofnaturalnos.py

### PR DESCRIPTION
**Pull request by yashkumarjha12**

### CODE REVIEW

**Comment by programmer-offbeat** [comment]
Thanks so much for the contribution. Make sure to add a star :)

File: Python/sumofnaturalnos.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    if num < 0:
+       print("Enter a positive number")
+    else:
+       sum = 0
+       # use while loop to iterate until zero
+       while(num > 0):
+           sum += num
+           num -= 1
+       print("The sum is", sum)
+
```
Comment (ShubhamJagtap2000) [inline Python/sumofnaturalnos.py:11]: You can add another method as a comment. Simple maths formula of finding the sum of first n natural numbers as n(n+1)/2 in py.

```py
# Another Method

def findSum(num) :
    return num * (num + 1) / 2
n = int(input())
print findSum(n)
```

Comment (ShubhamJagtap2000) [reply Python/sumofnaturalnos.py:11]: You can wish to add this method as comment if you want to give more insight to beginners.

Comment (ShubhamJagtap2000) [reply Python/sumofnaturalnos.py:11]: Please review or you can also cancel the changes made by me if they are not necessary for the code.

Comment (programmer-offbeat) [reply Python/sumofnaturalnos.py:11]: You can create an issue and work on it, I'll merge them:)

Comment (ShubhamJagtap2000) [reply Python/sumofnaturalnos.py:11]: Okay. will do.

**Pull request merged**